sotodlib/core/metadata/obsdb.py

```diff
@@ -1,19 +1,20 @@
-import sqlite3
 import os
+import psycopg
+from typing import Optional, Union
 
 from .resultset import ResultSet
 from . import common
 
 
 TABLE_DEFS = {
-    'obs': [
-        "`obs_id` varchar(256) primary key",
-        "`timestamp` float",
+    "obs": [
+        "obs_id varchar(256) primary key",
+        "timestamp real",
     ],
-    'tags': [
-        "`obs_id` varchar(256)",
-        "`tag` varchar(256)",
-        "CONSTRAINT one_tag UNIQUE (`obs_id`, `tag`)",
+    "tags": [
+        "obs_id varchar(256)",
```

Comment:
obs_id is not a foreign key?

Reply:
It seems it is not. I think it should be and it will make search and joins even faster